packages/code-studio/src/dashboard/linker/Linker.tsx

```diff
@@ -152,7 +222,7 @@ class Linker extends Component {
       link => link.start?.panelId === panelId || link.end?.panelId === panelId
```

Comment:
My VSCode is yelling at me about this being implicitly any. I think it's something to do with the `ConnectedProps` in `LinkerProps` because when I remove that this (and 4 or 5 other spots) have proper types instead of any

We don't have a linter rule and I'm not sure that one exists for it. I found some that are close. They require a bit of config w/ eslint since you have to tell it about the TS runtime so it can do type analysis